tools/src/versioning/ios/transforms/vendoredModulesTransforms.ts

```diff
@@ -23,6 +23,16 @@ export default function vendoredModulesTransformsFactory(prefix: string): Config
           find: /RCT_EXTERN_MODULE\(/,
           replaceWith: `RCT_EXTERN_REMAP_MODULE(AuBECSDebitFormManager, ${prefix}`,
         },
+        {
+          paths: 'StripeSdk.m',
+          find: /RCT_EXTERN_MODULE\(/,
+          replaceWith: `RCT_EXTERN_REMAP_MODULE(StripeSdk, ${prefix}`,
+        },
+        {
+          paths: 'StripeContainerManager.m',
+          find: /RCT_EXTERN_MODULE\(/,
+          replaceWith: `RCT_EXTERN_REMAP_MODULE(StripeContainerManager, ${prefix}`,
+        },
```

Comment:
Maybe we can simplify them all to 👇?
```suggestion
        {
          paths: '*.m',
          find: /RCT_EXTERN_MODULE\((ApplePayButtonManager|CardFieldManager|AuBECSDebitFormManager|StripeSdk|StripeContainerManager)/,
          replaceWith: `RCT_EXTERN_REMAP_MODULE($1, ${prefix}$1`,
        },
```

Reply:
good thinking 👍 tested and works well